tests/pkg-layering/main.yml

```diff
@@ -310,7 +318,10 @@
 
     - name: Fail if error not expected
       fail:
-        msg: "Command error not expected"
+        msg: |
+          Expected: {{ g_invalid_pkg }} is not currently requested in rpm-ostree
+                    uninstall output
+          Actual: {{ uninstall.stderr }}
       when: "'\\'{{ g_invalid_pkg }}\\' is not currently requested' not in uninstall.stderr"
```

Comment:
As an aside unrelated to the changes here, this conditional looks ugly as sin.  I wonder if we can improve it....